include/view.h

```diff
@@ -106,6 +106,13 @@ struct xdg_toplevel_view {
 	struct wl_listener new_popup;
 };
 
+static inline bool
+view_is_floating(struct view *view)
+{
+	return !view->fullscreen && !view->maximized && !view->tiled
+		&& !view->tiled_region;
```

Comment:
This is missing a check for `view->tiled_region_evacuate`.
Alternative: `!view->fullscreen && !view->maximized && !view_is_tiled(view)`.

I think this should be a function in `src/view.c` instead. I don't think we have any other functions in headers.

Reply:
Good catch on tiled_region_evacuate. I'm not to familiar with the new region code.

I think I saw one other static inline function in a header but I don't feel strongly one way or the other on that point.

Want to create a PR to address the points you mentioned?

Reply:
Sure, can do later